package.json

```diff
@@ -1,15 +1,13 @@
 {
   "private": true,
   "scripts": {
-    "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "eslint src && pulp build -- --censor-lib --strict",
-    "test": "pulp build -I example/src"
+    "build": "eslint src && spago build --purs-args '--censor-lib --strict'",
+    "test": "spago test --no-install"
   },
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "pulp": "^12.2.0",
-    "purescript": "slamdata/node-purescript#0.12",
-    "purescript-psa": "^0.6.0",
-    "rimraf": "^2.6.2"
+    "eslint": "^7.6.0"
+  },
+  "dependencies": {
+    "ace-builds": "^1.4.12"
```

Comment:
I see this has moved from the `bower.json` file. That's good -- moving from Bower requires that and good catch! However, I believe that means the `index.html` file in the `examples` directory will also need to be updated to point at the correct location. Also, the repo README should be updated to note that you also need to install `ace-builds` to get the editor. For example:

> You will also need to install the Ace editor. For example, to install it from NPM:
> 
> ```sh
> npm install --save ace-builds
> ```

Reply:
`--save` option is [no-longer required](https://stackoverflow.com/a/19578808).